panel/src/panel/panel.js

```diff
@@ -91,6 +90,7 @@ export default {
 		// features
 		this.dropdown = Dropdown(this);
 		this.view = View(this);
+		this.content = Content(this);
```

Comment:
`Comment:` without this I sometimes got console errors on first load for not finding `panel.view.props` etc.

Reply:
I'm ok with that.